Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,442 @@
+# Migrating from Analytics mobile extension to Edge Network using the EdgeBridge extension
+## Overview
+This hands-on tutorial provides end-to-end instructions on how to migrate to Edge Network from Analytics to EdgeBridge mobile extension.
+
+There are other beginning states the customer can be in like using ACP Analytics, v4 extension, etc. These usages can potentially be migrated to AEP and from there, this tutorial can be applied afterwards. However, this should be determined on a case-by-case basis, as for example in the v4 extension case, it may be more effective to simply implement AEP extensions itself without the need for Edge Bridge migration path.
+
+Tutorial setup:
+
+Initial test app that is based on prerequisites (implements Analytics)
+Final test app after all the tutorial steps have been implemented (implements EdgeBridge)
+
+Tutorial steps:
+
+      0. Prerequisites (set up Analytics report suite, mobile property, Assurance). List out required permissions for this tutorial: Analytics, Schema creation, Data Collection (Launch tags), Datastream view and edit, Assurance.
+
+## Data Collection config instructions
+Create XDM schema   
+Configure Datastream, enable Analytics - 2 paths:   
+    - same rsid(s) as in Analytics extension (if using Analytics + EdgeBridge this will cause double counting).  
+    - different rsid(s) if the customer wants to start new or run the migration in a comparison mode (Analytics + EdgeBridge side by side).  
+
+## Prerequisites
+- A timestamp enabled report suite is configured for mobile data collection.
+- A tag (also known as mobile property) is configured in Data Collection UI which has Adobe Analytics extension installed and configured.
+- AEPAnalytics mobile extension is installed and registered in the client-side mobile application.
+- `MobileCore.trackAction` and/or `MobileCore.trackState` API(s) are already implemented in the mobile application.
+
+### Environment
+- macOS machine with a recent version of Xcode installed
+
+Install Edge Network & Edge Identity in Launch - Edge Bridge does not have a card here  
```

Comment:
It is not obvious that what need to be done in Data Collection. 
How about take a screenshot of the required carts in the data collection Installed view for a property here.